### PR TITLE
IBX-8502: Added unsupported locale fallback

### PR DESCRIPTION
**Pull request by reithor**
| :ticket: Issue | IBX-8502 |
|----------------|-----------|

Issue seems to come from the fact that `Intl` does not support 3 letter region subtags:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl

```
A locale identifier is a string that consists of:
# A language subtag with 2–3 or 5–8 letters
# A script subtag with 4 letters Optional
# A region subtag with either 2 letters or 3 digits Optional (!)
# One or more variant subtags (all of which must be unique), each with either 5–8 alphanumerals or a digit followed by 3 alphanumerals Optional
# One or more BCP 47 extension sequences Optional
# A private-use extension sequence Optional
```
PR adds a fallback to `eng-GB` for locales not supported by `Intl`.

### CODE REVIEW

File: src/bundle/Resources/public/js/CKEditor/core/base-ckeditor.js
Comment (GrabowskiM) [inline src/bundle/Resources/public/js/CKEditor/core/base-ckeditor.js:132]: I would add some kind of `console.warn` to catch to give user any kind of message, that something has gone wrong - otherwise it's silent error and user having lang `ger` can be suprised why it's not working correctly and uses eng locale

**Review by GrabowskiM — CHANGES_REQUESTED** [review]

**Comment by dew326** [comment]
@reithor any progress on that?

**Comment by reithor** [comment]
> @reithor any progress on that?

@dew326 -  missed message from @GrabowskiM , will take care of that now.

File: src/bundle/Resources/public/js/CKEditor/core/base-ckeditor.js
Comment (reithor) [reply src/bundle/Resources/public/js/CKEditor/core/base-ckeditor.js:135]: I have added `console.warn` now.

**Review by GrabowskiM — APPROVED** [review]

**Review by dew326 — APPROVED** [review]

**Comment by reithor** [comment]
@dew326 this needs also approval from one maintainer ? @alongosz or @konradoboza ?

**Comment by dew326** [comment]
I'm the maintainer of this package as well. We are good. You can pass it to QA.

**Pull request merged**
